### PR TITLE
Replaced magic numbers with constants.

### LINKED ISSUE #23: Replace all Magic Numbers with Defined Constants
We have a lot of magic numbers in our subsystems right now. Please replace them with well-named constants.

### PR DESCRIPTION
**Pull request by AdamDaMiner**
Closes #23.

### CODE REVIEW

File: 2022 Season/src/main/java/frc/robot/Configuration/Constants.java [hunk 1]
```diff
@@ -13,15 +13,87 @@
     public static final int kLeftRearDriveDeviceId = 3;
     public static final int kRightFrontDriveDeviceId = 4;
     public static final int kRightRearDriveDeviceId = 5;
+    public static final int kHoodMotorDeviceId = 10; 
+    public static final int kHoodEncoderDeviceId = 4;
```
Comment (Koltara) [inline 2022 Season/src/main/java/frc/robot/Configuration/Constants.java:17]: Rename to kHoodEncoderPort

File: 2022 Season/src/main/java/frc/robot/Configuration/Constants.java [hunk 2]
```diff
@@ -13,15 +13,87 @@
...
     
+    // Talon FX Device Ids
+    public static final int kLeftShooterMotorDeviceId = 15; 
+    public static final int kRightShooterMotorDeviceId= 14;
+
+    // Left Shooter Motor Config Values
+    public static final double kLeftMotorKp= .015;
```
Comment (Koltara) [inline 2022 Season/src/main/java/frc/robot/Configuration/Constants.java:24]: Rename to kShooterKp

File: 2022 Season/src/main/java/frc/robot/Configuration/Constants.java [hunk 3]
```diff
@@ -13,15 +13,87 @@
...
+    public static final double kLeftMotorKf= .0473;
```
Comment (Koltara) [inline 2022 Season/src/main/java/frc/robot/Configuration/Constants.java:25]: Rename to kShooterKf

File: 2022 Season/src/main/java/frc/robot/Configuration/Constants.java [hunk 4]
```diff
@@ -13,15 +13,87 @@
...
+    // Hood Target Points
+    public static final double kHalfShotValue = 0.5;
+    public static final double kQuarterShotValue = 0.25;
+    public static final double kThreeQuarterShotValue =0.75;
```
Comment (Koltara) [inline 2022 Season/src/main/java/frc/robot/Configuration/Constants.java:29]: Add space.

File: 2022 Season/src/main/java/frc/robot/Configuration/Constants.java [hunk 5]
```diff
@@ -13,15 +13,87 @@
...
+    public static final double kFiveBallPartTwoFromTerminalEndRotation= -2.5878351309802463;
     //DIO
     public static final int kLeftEncoderPortA = 0;
     public static final int kLeftEncoderPortB = 1;
     public static final int kRightEncoderPortA = 2;
     public static final int kRightEncoderPortB = 3;
 
-    // Test Robot Configuration
+    //Hood PID Values
+    public static final double hoodKp= 4.5;
```
Comment (Koltara) [inline 2022 Season/src/main/java/frc/robot/Configuration/Constants.java:89]: kHoodKp

File: 2022 Season/src/main/java/frc/robot/Configuration/Constants.java [hunk 6]
```diff
@@ -13,15 +13,87 @@
...
     public static final int kRightEncoderPortB = 3;
 
-    // Test Robot Configuration
+    //Hood PID Values
+    public static final double hoodKp= 4.5; 
+    public static final double hoodKi= 0;
+    public static final double hoodKd= 0;
 
+    // Magnetic Limit Switch Id 
+    public static final int magneticLimitSwitchId = 5;
```
Comment (Koltara) [inline 2022 Season/src/main/java/frc/robot/Configuration/Constants.java:94]: Rename to kMagneticLimitSwitchPort

**Review by Koltara — CHANGES_REQUESTED** [review]

**Review by m11knu — APPROVED** [review]
looks all good from here

**Pull request merged**